tests/utilities_test.py

```diff
@@ -4,6 +4,7 @@
 import unittest
 
 import switch_mod.utilities as utilities
+import tempfile, os, shutil
```

Comment:
Suggestion: it can be useful to write one import per line, and to keep the imports sorted, i.e.

import os
import shutil
import tempfile

This reduces merge conflicts if two people independently add the same import (or it makes merge conflicts easier to resolve).